### PR TITLE
PEOPLEEAT-031: privacy-policy/terms-and-conditions links and open new…

### PR DESCRIPTION
**Pull request by Llavrov**
… window tab

### CODE REVIEW

File: src/components/pages/cookSignUp/index.tsx [hunk 1]
```diff
@@ -333,7 +333,12 @@ export default function CookSignUpPage({ signedInUser, languages }: CookSignUpPa
                         <FormControlLabel
                             control={<PECheckbox checked={acceptedPrivacyPolicy} onCheckedChange={setAcceptedPrivacyPolicy} />}
                             label={
-                                <Link className="no-underline text-orange" href={'/data-privacy-policy'}>
+                                <Link
+                                    className="no-underline text-orange"
+                                    href={'/data-privacy-policy'}
+                                    rel="noopener noreferrer"
```
Comment (CemYil03) [inline src/components/pages/cookSignUp/index.tsx:339]: Gould you explain what `rel="noopener noreferrer"` is supposed to do? I have a feeling it doesn't change the behaviour

Comment (Llavrov) [reply src/components/pages/cookSignUp/index.tsx:339]: It's open a new tab in brawser

File: locales/fr/chef-sign-up.json [hunk 2]
```diff
@@ -16,9 +16,13 @@
     "phone-number": "Numéro de téléphone",
     "post-code": "Code postal",
     "privacy-policy-label": "J'ai lu et j'accepte la politique de confidentialité",
+    "privacy-policy-label-link": "politique de confidentialité",
+    "privacy-policy-label-2": "",
     "street": "Rue",
     "sub-headline": "Veuillez saisir vos coordonnées",
     "terms-and-conditions-label": "J'ai lu et j'accepte les termes et conditions",
```
Comment (CemYil03) [inline locales/fr/chef-sign-up.json:23]: This look wrong
```suggestion
    "terms-and-conditions-label": "J'ai lu et j'accepte les",
```

**Review by CemYil03 — CHANGES_REQUESTED** [review]
Both Booking Request Pages are missing. In the task they are all listed...

**Pull request merged**